### PR TITLE
Update mailing list for gci jobs

### PR DESCRIPTION
**Pull request by wonderfly**
@adityakali @Amey-D Can you review?

### CODE REVIEW

**Comment by Amey-D** [comment]
LGTM

**Comment by wonderfly** [comment]
I am going to hand merge this as it's just a mailing list change, and we want to stop spamming the developer mailing list asap.

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml [hunk 1]
```diff
@@ -65,7 +65,7 @@
     # Template defaults.
     jenkins_node: 'e2e'
     test-owner: 'wonderfly'
-    emails: 'wonderfly@google.com,qzheng@google.com,lakitu-dev@google.com'
+    emails: 'gci-alerts+kubekins@google.com'
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: this isn't going to work unless the mailing list allows posting from outside google.com.

Comment (wonderfly) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml]: Right. We had to open it to public.

**Pull request merged**